### PR TITLE
Ddpb 1856 - professional deputies

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: app/config/security.yml [hunk 1]
```diff
@@ -101,10 +101,22 @@ security:
...
+        ROLE_ORG:             ROLE_DEPUTY
+
+        ROLE_PA:              ROLE_ORG
+        ROLE_PROF:            ROLE_ORG
+
+        ROLE_PA_NAMED:        [ ROLE_PA, ROLE_ORG_NAMED ]
+        ROLE_PA_ADMIN:        [ ROLE_PA, ROLE_ORG_ADMIN ]
+        ROLE_PA_TEAM_MEMBER:  [ ROLE_PA, ROLE_ORG_TEAM_MEMBER ]
+
+        ROLE_PROF_NAMED:        [ ROLE_PA, ROLE_ORG_NAMED ]
```
Comment (slizzio) [inline app/config/security.yml]: should ROLE_ORG_NAMED be specified somewhere?

File: src/AppBundle/Controller/Ndr/AccountController.php [hunk 2]
```diff
@@ -5,19 +5,18 @@
 use AppBundle\Controller\RestController;
 use AppBundle\Entity as EntityDir;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
```
Comment (slizzio) [inline src/AppBundle/Controller/Ndr/AccountController.php]: new line

File: src/AppBundle/Controller/Ndr/AssetController.php [hunk 3]
```diff
@@ -5,19 +5,18 @@
 use AppBundle\Controller\RestController;
 use AppBundle\Entity as EntityDir;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
```
Comment (slizzio) [inline src/AppBundle/Controller/Ndr/AssetController.php]: new line

File: src/AppBundle/Controller/Ndr/ExpenseController.php [hunk 4]
```diff
@@ -5,19 +5,18 @@
 use AppBundle\Controller\RestController;
 use AppBundle\Entity as EntityDir;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
```
Comment (slizzio) [inline src/AppBundle/Controller/Ndr/ExpenseController.php]: new line

File: src/AppBundle/Controller/Ndr/NdrController.php [hunk 5]
```diff
@@ -5,7 +5,7 @@
 use AppBundle\Controller\RestController;
 use AppBundle\Entity as EntityDir;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
```
Comment (slizzio) [inline src/AppBundle/Controller/Ndr/NdrController.php]: new line

File: src/AppBundle/Controller/Ndr/VisitsCareController.php [hunk 6]
```diff
@@ -5,19 +5,18 @@
 use AppBundle\Controller\RestController;
 use AppBundle\Entity as EntityDir;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
```
Comment (slizzio) [inline src/AppBundle/Controller/Ndr/VisitsCareController.php]: new line

File: app/config/security.yml (hunk 1 above)
Comment (elvisciotti) [reply app/config/security.yml]: already discussed

**Pull request merged**